Devscord.DiscordFramework/ControllersService.cs

```diff
@@ -141,10 +141,6 @@ private async Task RunMethodsIBotCommand(DiscordRequest request, Contexts contex
                         {
                             command = this._botCommandsService.ParseRequestToCommand(commandInParameterType, request, template);
                         }
-                        if (!this.IsValid(contexts, method))
```

Comment:
to ma właśnie zostać
bo bo co sprawdzać dla wszystkich komend czy są uprawnienia
przez to rzuca do każdej komendy wyjątki jeśli user nie jest adminem